config/config.go

```diff
@@ -1020,8 +1026,16 @@ func SetupViper(v *viper.Viper, filename string, bidderInfos BidderInfos) {
 	v.SetDefault("account_defaults.price_floors.use_dynamic_data", false)
 	v.SetDefault("account_defaults.price_floors.max_rules", 100)
 	v.SetDefault("account_defaults.price_floors.max_schema_dims", 3)
-	v.SetDefault("account_defaults.events_enabled", false)
+	v.SetDefault("account_defaults.price_floors.fetch.enabled", false)
+	v.SetDefault("account_defaults.price_floors.fetch.timeout_ms", 100)
+	v.SetDefault("account_defaults.price_floors.fetch.period_sec", 300)
+	v.SetDefault("account_defaults.price_floors.fetch.max_age_sec", 600)
+
+	//Defaults for Price floor fetcher
+	v.SetDefault("price_floor.fetcher.worker", 20)
+	v.SetDefault("price_floor.fetcher.capacity", 20000)
```

Comment:
How were these default values decided? I couldn't find them in the floors documentation

Reply:
20000 will be large number if prebid is hosted by a publisher, as he may have maximum 2-3 urls, but if we think about hosting company who are working with multiple company then this number will be valid in maximum case
We can make this configurable, through configuration file.